packages/components/src/server-side-render/index.js

```diff
@@ -44,10 +44,15 @@ export class ServerSideRender extends Component {
 		this.isStillMounted = false;
 	}
 
-	componentDidUpdate( prevProps ) {
+	componentDidUpdate( prevProps, prevState ) {
 		if ( ! isEqual( prevProps, this.props ) ) {
 			this.fetch( this.props );
 		}
+		if ( this.state.response !== prevState.response ) {
```

Comment:
It's a bit awkward to call a prop in response to a state change in `componentDidUpdate`, since a component manages its own state and begs the question why isn't the prop called at the time we're calling `setState` ? Our current implementation of `ServerSideRender#fetch` doesn't make this very easy to refactor, though...

Reply:
Is this a merge blocker? If not can we bring it to an issue after? I think there would have to be an alteration of ServerSideRender to accomplish this. I fear it may introduce more bugs